src/gui/DatabaseWidget.cpp

```diff
@@ -333,6 +334,18 @@ void DatabaseWidget::cloneEntry()
     return;
 }
 
+void DatabaseWidget::getTotp()
+{
+    Entry* currentEntry = m_entryView->currentEntry();
+    if (!currentEntry) {
+        Q_ASSERT(false);
+        return;
+    }
+
+    TotpDialog* totpDialog = new TotpDialog(this, currentEntry);
```

Comment:
@rockihack @phoerious 
I had `Qt::WA_DeleteOnClose` already set at [TotpDialog.cpp line 34](https://github.com/weslly/keepassxc/blob/feature/totp/src/gui/TotpDialog.cpp#L34) before, doesn't it have the same effect?